frontend/src/app/questPage/Referral/page.tsx

```diff
@@ -0,0 +1,30 @@
+"use client";
```

Comment:
wrong place, put in /app/quests/referral

also , i dont see the sidebar component?

Reply:
The page layout was already defined. The Sidebar and Nav bar components were imported there already.